quickwit/quickwit-control-plane/src/control_plane.rs

```diff
@@ -71,13 +72,16 @@ pub(crate) const CONTROL_PLAN_LOOP_INTERVAL: Duration = if cfg!(any(test, featur
     Duration::from_secs(5)
 };
 
+/// Minimum period between two identical shard pruning operations.
+const PRUNE_SHARDS_DEFAULT_COOLDOWN_PERIOD: Duration = Duration::from_secs(120);
+
 /// Minimum period between two rebuild plan operations.
 const REBUILD_PLAN_COOLDOWN_PERIOD: Duration = Duration::from_secs(2);
 
 #[derive(Debug)]
 struct ControlPlanLoop;
 
-#[derive(Debug, Default)]
+#[derive(Debug, Default, Clone)]
```

Comment:
```suggestion
#[derive(Debug, Default, Clone, Copy)]
```

Reply:
I don't see any harm in implementing Copy here, but what is the actual benefit?

Reply:
nevermind, irrelevant if considering [this comment](https://github.com/quickwit-oss/quickwit/pull/5374#discussion_r1764588527)